src/v2/models/AppState.js

```diff
@@ -157,6 +157,16 @@ export default Model.extend({
       reRender = false;
     }
 
+    if (identicalResponse && this.get('currentFormName') === 'poll') {
```

Comment:
there is one more PR.. may have a conflict..

https://github.com/okta/okta-signin-widget/pull/1795/files#r605108218